.php-cs-fixer.dist.php

```diff
@@ -42,6 +42,7 @@
     'multiline_whitespace_before_semicolons' => ['strategy' => 'no_multi_line'],
     'no_useless_else' => true,
     'no_useless_return' => true,
+    'no_superfluous_phpdoc_tags' => ['allow_mixed' => true, 'remove_inheritdoc' => true],
```

Comment:
This will be part of dev-kit: https://github.com/sonata-project/dev-kit/pull/2468